### ISSUE TITLE
Advance Date button skipping 1 month rather than 1 day.

### ISSUE THREAD

**Issue by tdadvocate** [LegoFigure11/RaidCrawler#151]
**Describe the bug**
Advance Date button skipping 1 month rather than 1 day.

**To Reproduce**
If applicable, please list the steps to reproduce the behavior:
1: Click "Open Settings"
2: Go to "Advance Date"
3: Set "Number of days/months to skip:" to 1
4: Go back to the main GUI screen
5: Click on "Advance Date"
6: The system will now skip ahead 1 month rather than 1 day.

**Expected behavior**
This should advance the date by 1 day instead of 1 month to allow for the most amount of checks for the raids that the user is looking for.

**Additional context**
Switch Firmware 16.0.3
Atmosphere Version 1.5.4
Hekate Version 6.0.5
Nyx Version 1.5.4
Scarlet Version 1.3.1
RaidCrawler Version 1.3.2

**Comment by tdadvocate** [comment]
I just realised that this could be an issue with the format of the date on US systems. It looks like this may be set up for a region that uses Day/Month/Year format instead of the Month/Day/Year format like the US. I think we would need a whole new option just to change this without users having to change their date formats on the Switch itself. Or you should mention that this is needed for US systems in the tutorial for us jerks who don't use a format that makes sense.

**Comment by Koi-3088** [comment]
Any date/time manipulation should result in a single skip. In fact, you don't even need to adjust anything - opening the menu and confirming is enough.

**Comment by tdadvocate** [comment]
> Any date/time manipulation should result in a single skip. In fact, you don't even need to adjust anything - opening the menu and confirming is enough.

I was not aware of this. I was wondering why it was defaulted to 0. So does that mean that the best way to do this would be just default settings on Account 1 and then connect with account 2 whenever the shiny raid is found? Would using the same save file cause conflicts if it was loaded on 2 different accounts?

**Comment by LegoFigure11** [comment]
> I was not aware of this. I was wondering why it was defaulted to 0. So does that mean that the best way to do this would be just default settings on Account 1 and then connect with account 2 whenever the shiny raid is found? Would using the same save file cause conflicts if it was loaded on 2 different accounts?

I'm not sure I understand the question? Multiple accounts is entirely unnecessary, and you can just leave it at 0 (default).

**Comment by tdadvocate** [comment]
> I'm not sure I understand the question? Multiple accounts is entirely unnecessary, and you can just leave it at 0 (default).
> 
I have set it back to 0 as I didn't realise this was necessary. I have multiple accounts for events and whatnot. So, I was just curious how it all worked. I guess the more important question would be if I were to save after finding a shiny raid, could I move the save from my CFW console to my non-CFW console, or would that reset the seed? 

Unrelated and sorry for asking here, but I am not sure where else to ask. How does one connect via USB? I figured it might be more reliable than via Internet but can't find any info on this feature.
